storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/queries/Query.java

```diff
@@ -477,6 +477,50 @@ public CompleteBuilder limit(@Nullable String limit) {
         }
 
         /**
+         * Optional: Specifies {@code LIMIT} clause.
+         * <p>
+         * Optional specifier that limits the number of rows returned by the query.
+         *
+         * @param limit positive number of rows returned by the query.
+         * @return builder.
+         * @see Query#limit()
+         */
+        @NonNull
+        public CompleteBuilder limit(final int limit) {
+            if (limit <= 0) {
+                throw new IllegalStateException("Limit should have a positive value");
+            }
+            this.limit = String.valueOf(limit);
+            return this;
+        }
+
+        /**
+         * Optional: Specifies {@code LIMIT} clause.
+         * <p>
+         * Optional specifier that limits the number of rows returned by the query.
+         * <p>
+         * Examples:
+         * <ul>
+         * <li>{@code "from = 5, quantity = 12"} — will limit output to 12 rows with start offset 5.</li>
+         *
+         * @param from non-negative start offset.
+         * @param quantity positive number of queried rows.
+         * @return builder.
+         * @see Query#limit()
+         */
+        @NonNull
+        public CompleteBuilder limit(final int from, final int quantity) {
```

Comment:
I guess, `from` should be renamed to `offset`? Official documentation uses term `offset` https://www.sqlite.org/lang_select.html#limitoffset